content/blog/2021/2021-12-17-visualizations-101.md

```diff
@@ -0,0 +1,77 @@
+
+**How to read the plot**: *thal* is the variable that best separates between patients with and without diameter narrowing. There is a higher incidence of reversable defect in patients with diameter narrowing (64% have reversable defect) that in those without diameter narrowing (17% have reversable defect).
+
+### Distributions
+
+Distributions should probably be named histogram, as it displays a histogram of selected variable. Numeric variables will be discretized - this means that Orange will create bins, where values within a certain range will be put. In the below example, there are two bins, one with values below 50.87 and one with values above 50.78.
+
+{{< window-screenshot src="/blog_img/2021/2021-12-17-distributions.png" >}}
+
+The widget can be used to display differences in distributions.
+
+**How to read the plot**: In the group of patients that are younger than 50 years, there is a 50% lower presence of diameter narrowing (70 % of patients don't have the condition). 
+
+### Sieve Diagram
+
+Sieve diagram is a complex, yet extremely useful visualization. It shows the difference between the expected and observed variable frequency. In other words, if a combination of two values, i.e. being 60 years old and having a heart disease, is more frequent than expected, Sieve diagram will show it.
+
+{{< window-screenshot src="/blog_img/2021/2021-12-17-sieve.png" >}}
+
+In the scatter plot above, we observed a slightly higher incidence of diameter narrowing in patient around 60 years of age. Let us check this is Sieve diagram.
+
+On y axis we see *diameter narrowing*, with the size of each box vertically corresponding to the proportion of instances with and without condition. On x axis we see *age*, with the size of each box horizontally corresponding to the size of each bin. As numeric variables are discretized with equal frequency discretization, the box will always be the same size.
+
+The color of the box corresponds to how overrepresented (blue) or underrepresented (red) a certain combination is. For example, there are 73 patients (24 %) between 55.5 and 60.5 years old in the data set. There are also 139 patients (46%) with diameter narrowing.
+
+If these two variables would be unrelated, we would expect around 11% of patients with both values (0.24 x 0.46 = 0.11), that is patients between 55 and 60 years old with a diameter narrowing.
+
+But if we observe the data set, we can see there are actually 15% of such patients. That is more than we would expect, so a combination of the two values could be statistically significant.
+
+In the left corner of the plot we once again meet the Chi-square, which tell us how significant the combination actually is.
```

Comment:
meet the Chi-squared statistic, which tells us how significant the combination actually is.